source/layout.erb

```diff
@@ -31,19 +31,17 @@
       <%= partial "sidebar" %>
 
       <article>
-        <div class="chapter">
```

Comment:
@josemarluedke Other than the `.chapter .li`, I'm not seeing any other reliance on `.chapter`, have I missed anything?